app/src/main/java/com/danlls/daniel/todule_android/activities/TimeReminderFragment.java

```diff
@@ -0,0 +1,170 @@
+package com.danlls.daniel.todule_android.activities;
+
+
+import android.app.TimePickerDialog;
+import android.content.DialogInterface;
+import android.content.SharedPreferences;
+import android.os.Bundle;
+import android.preference.PreferenceManager;
+import android.support.design.widget.TextInputEditText;
+import android.support.v4.app.Fragment;
+import android.text.Editable;
+import android.text.InputType;
+import android.text.TextWatcher;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.Menu;
+import android.view.MenuInflater;
+import android.view.MenuItem;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.TimePicker;
+import android.widget.Toast;
+
+import java.util.Calendar;
+
+import com.danlls.daniel.todule_android.R;
+
+
+public class TimeReminderFragment extends Fragment {
+    SharedPreferences sf;
+    Calendar myCalendar;
+    MainActivity myActivity;
+    TextInputEditText timeViewHours;
+    TextInputEditText timeViewDays;
+
+
+    public static TimeReminderFragment newInstance() {
+
+        TimeReminderFragment fragment = new TimeReminderFragment();
+        return fragment;
+    }
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        sf = PreferenceManager.getDefaultSharedPreferences(getContext());
+        myCalendar = Calendar.getInstance();
+        setHasOptionsMenu(true);
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        myActivity = (MainActivity) getActivity();
+        myActivity.getSupportActionBar().setTitle("Edit Reminder");
+        myActivity.hideSoftKeyboard(true);
+
+        // Inflate the layout for this fragment
+        View view = inflater.inflate(R.layout.fragment_time_reminder, container, false);
+        timeViewDays = view.findViewById(R.id.timeViewDays);
+        timeViewHours = view.findViewById(R.id.timeViewHours);
+        //timeViewDays.setInputType(InputType.TYPE_NUMBER_FLAG_DECIMAL);
+        timeViewHours.setInputType(InputType.TYPE_NULL);
+
+        // Default for new entry
+        if (sf.contains("timing")){
+            myCalendar.set(Calendar.HOUR_OF_DAY, sf.getInt("hours",1));
+            myCalendar.set(Calendar.MINUTE, sf.getInt("minutes",0));
+            myCalendar.set(Calendar.SECOND, 0);
+            timeViewHours.setText(String.valueOf(sf.getInt("days",0)));
+        }else {
+            myCalendar.set(Calendar.HOUR_OF_DAY, 1);
+            myCalendar.set(Calendar.MINUTE, 0);
+            myCalendar.set(Calendar.SECOND, 0);
+            timeViewHours.setHint("0 Days");
```

Comment:
Typo: `timeViewHours` should be replaced by `timeViewDays`. I don't think hint is required here as you added `TextView`, set default value of `timeViewDays` to 0 instead.